gallery/visualtests/nxCheckbox.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { clickTest, focusTest, focusAndHoverTest, hoverTest, simpleTest } = require('./testUtils');
+
+describe('NxCheckbox', function() {
+  beforeEach(async function() {
+    await browser.url('#/pages/NxCheckbox');
+  });
+
+  describe('Default NxCheckbox', function() {
+    const selector = '#nx-checkbox-default-example';
+
+    it('has a light grey border and white background by default', simpleTest(selector));
+    it('has a black border when hovered', hoverTest(selector));
+    it('has a thick blue border and white background when clicked', clickTest(selector));
```

Comment:
Note that the click test takes the screenshot while the mouse is depressed.  If you want to fully click it and _then_ take the screenshot, you'll need to write something more custom.

Reply:
> Note that the click test takes the screenshot while the mouse is depressed. If you want to fully click it and _then_ take the screenshot, you'll need to write something more custom.

That might be a bit beyond me right now